spec/10-expressions.md

```diff
@@ -2796,7 +2796,7 @@ $a = new C; // make $a point to the allocated object
 *unary-expression* must designate a variable.
 
 *assignment-expression* must be an lvalue, a call to a function that
-returns a value byRef, or a *new-expression* (see comment below
+returns a value byRef, or a *new-expression* that does not have the form `new class` (see comment below
```

Comment:
We've dropped support for assigning new by reference in PHP 7, so this can be dropped altogether.